submissions/IhorOzerov/html-css-popup/index.html

```diff
@@ -0,0 +1,156 @@
+                            <img class="visible-img" src="icons/contacts.png"alt="contacts">
+                            <span>Contacts</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="menu-a" href="#"> 
+                            <img class="visible-img" src="icons/documents.png" alt="documents">
+                            <span>Documents</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="menu-a" href="#">
+                            <img class="visible-img" src="icons/files.png"alt="files">
+                            <span>Files</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="menu-a" href="#"> 
+                            <img class="visible-img" src="icons/mail.png"alt="mail">
+                            <span>Mail</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="menu-a" href="#"> 
+                            <img class="visible-img" src="icons/maps.png"alt="maps">
+                            <span>Maps</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="menu-a" href="#"> 
+                            <img class="visible-img" src="icons/mars.png"alt="mars">
+                            <span>Mars</span>
+                        </a>
+                    </li>
+                    <li class="more-btn greatest">
+                        <label class="more-btn" for="more">More</label>
+                        <input id="more" type="checkbox" class="visually-hidden">
+                        <ul class="opened">
+                            <li>
+                                <a class="menu-a" href="#"> 
+                                    <img class="visible-img" src="icons/news.png"alt="news">
+                                    <span>News</span>
+                                </a>
+                            </li>
+                            <li>
+                                <a class="menu-a" href="#">
+                                    <img class="visible-img" src="icons/photos.png"alt="photos">
+                                    <span>Photos</span>
+                                </a>
+                            </li>
+                            <li>
+                                <a class="menu-a" href="#"> 
+                                    <img class="visible-img" src="icons/promo.png"alt="promo">
+                                    <span>Promo</span>
+                                </a>
+                            </li>
+                            <li>
+                                <a class="menu-a" href="#"> 
+                                    <img class="visible-img" src="icons/search.png" alt="search">
```

Comment:
If you use `img` tag for icons you should leave an empty value for `alt` attribute